### DISCUSSION TITLE
大数据量场景下的初次渲染性能问题

### DISCUSSION THREAD

**Discussion by wygkzqa** [alibaba/formily#3168] [Q&A 1.x] [answered]
我们的场景是一个问卷系统，有各种题目类型，添加了不到 100 道题目后，初次渲染卡顿的就非常厉害，不知道在 1.x 版本是否存在这种性能问题以及如何应对？谢谢，希望能解答一下

<img width="1679" alt="image" src="https://user-images.githubusercontent.com/7590234/172165385-2e7aa52b-5e7c-4178-84eb-bd67a752c3d3.png">

**Comment by janryWang** [comment, marked as answer]
这是1.x？2.0不会这么卡的，2000个字段渲染2秒左右

**Reply by wygkzqa** [reply]
对的，我们还用的 1.x

**Reply by wygkzqa** [reply]
除了升到 2.0，我们要解决性能有其他办法吗？
